commodore/dependency_mgmt.py

```diff
@@ -243,6 +254,58 @@ def fetch_jsonnet_libraries(cwd: P = None):
             "the jsonnet-bundler executable `jb` could not be found"
         ) from e
 
+    # Link essential libraries for backwards compatibility.
```

Comment:
How is this used? As long as the search path is correct the library should be discovered, no?

Reply:
No. We include those libraries as `lib/<name>.libjsonnet`. Without those links, we would need to rewrite all components to use `<library name>/<file name>`.